tests/python_functional/functional_tests/source_drivers/network_source/proxyprotocol/test_pp_acceptance.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import src.testcase_parameters.testcase_parameters as tc_parameters
+from src.common.operations import create_file
+
+TEMPLATE = r'"${PROXIED_SRCIP} ${PROXIED_DSTIP} ${PROXIED_SRCPORT} ${PROXIED_DSTPORT} ${PROXIED_IP_VERSION} ${MESSAGE}\n"'
+INPUT_MESSAGES = "PROXY TCP4 1.1.1.1 2.2.2.2 3333 4444\r\n" \
+                 "message 0"
+EXPECTED_MESSAGE0 = "1.1.1.1 2.2.2.2 3333 4444 4 message 0\n"
+
+
+def test_pp_acceptance(config, syslog_ng, loggen, port_allocator):
+    loggen_input_path = str(tc_parameters.WORKING_DIR) + "/loggen_input.txt"
+
+    network_source = config.create_network_source(ip="localhost", port=port_allocator(), transport='"proxied-tcp"', flags="no-parse")
+    file_destination = config.create_file_destination(file_name="output.log", template=TEMPLATE)
+    config.create_logpath(statements=[network_source, file_destination])
+
+    syslog_ng.start(config)
+
+    create_file(loggen_input_path, INPUT_MESSAGES)
```

Comment:
This is just a note, not a request.
This pattern emerges, a loggen must send user defined format toward a network source.

```python
loggen_input_path = str(tc_parameters.WORKING_DIR) + "/loggen_input.txt"
create_file(loggen_input_path, INPUT_MESSAGES)
loggen.start(network_source.options["ip"], network_source.options["port"], stream=True, permanent=True, read_file=loggen_input_path, dont_parse=True)
```

where stuff like where the file is created or the fact that there is a file unimportant. maybe in the future loggen could have an api for sanding raw data toward a network interface

```python
loggen.send_raw_input(network_source.options["ip"], network_source.options["port"], stream=True,  content=INPUT_MESSAGES)
```

That would also enable things like implementing a loggen option to provide input directly without a file, and the interface could act accordingly in light.